contracts/button/lib.rs

```diff
@@ -56,6 +60,12 @@ pub mod button_game {
         when: BlockNumber,
     }
 
+    #[ink(event)]
+    pub struct Halted {}
```

Comment:
In genera, Rust allows for "unit structs":
```rust
struct Halted;
```
and when constructing it's just `Halted`. Can we do it?

Reply:
needs literal syntax when instantiating. macros

Reply:
because that's not ink! DSL